### PR TITLE
Box sets (and a couple of forgotten booster boxes)

### PR DESCRIPTION
**Pull request by kodawah**
depends on #44 and on https://github.com/taw/magic-preconstructed-decks/pull/71

### CODE REVIEW

File: data/contents/CED.yaml
Comment (axxroytovu) [inline data/contents/CED.yaml:1]: You should add `card_count` to these.

File: data/contents/CEI.yaml
Comment (axxroytovu) [inline data/contents/CEI.yaml:1]: You should add `card_count` to these.

File: data/contents/CED.yaml
Comment (kodawah) [reply data/contents/CED.yaml:1]: updated!

File: data/contents/CEI.yaml
Comment (kodawah) [reply data/contents/CEI.yaml:1]: updated!

**Comment by kodawah** [comment]
pr updated and rebased

**Pull request merged**